playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -64,6 +65,7 @@ class RelatedPod(BaseModel):
     addresses: str
     containers: List[RelatedContainer]
     status: Optional[str] = None
+    status_reason: Optional[str] = None
```

Comment:
Can you please change to camel case to be consistent with the other class fields?

Reply:
done